compiler/src/iree/compiler/Codegen/Common/BUILD

```diff
@@ -32,6 +32,60 @@ iree_gentbl_cc_library(
     ],
 )
 
+# TODO: If the layering causes concerns then the transform dialect interpreter
```

Comment:
Willing to see where this goes for the moment. It being kind of a swallow the world of all deps is a bit troubling and we may want to find a way to... do something else clever. Fine for now.

Reply:
right that is the potential concern.